### PR TITLE
Mammoth Tank Proposed Changes

### PR DESCRIPTION
**Pull request by Jundiyy**
Proposing changes for the Mammoth Tank to make it unique rather than being the same as the Overlord Tank.

Maybe commy can add a small flame to the shell? It feels like it could have something like that, shouldn't do insane damage but just burn a bit.

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/CivilianUnit.ini [hunk 1]
```diff
@@ -269,8 +269,8 @@ Object MammothTank
   KindOf                 = PRELOAD SELECTABLE CAN_ATTACK ATTACK_NEEDS_LINE_OF_SIGHT CAN_CAST_REFLECTIONS VEHICLE SCORE
 
   Body                   = ActiveBody ModuleTag_02
-    MaxHealth       = 500
-    InitialHealth   = 500
+    MaxHealth       = 900
+    InitialHealth   = 900
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/CivilianUnit.ini]: SubdualDamageCap to 1800

Comment (Jundiyy) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/CivilianUnit.ini]: Oh yea, good catch.

**Comment by Jundiyy** [comment]
Done.

**Review by xezon — APPROVED** [review]

**Comment by xezon** [comment]
One more question: Are all loca strings for Mammoth Tank already in place?

**Pull request closed without merging**